vendor/src/github.com/docker/engine-api/types/events/events.go

```diff
@@ -9,8 +9,14 @@ const (
 	VolumeEventType = "volume"
 	// NetworkEventType is the event type that networks generate
 	NetworkEventType = "network"
-	// DaemonEventType is the event type that daemon generate
+	// DaemonEventType is the event type that daemon generates
 	DaemonEventType = "daemon"
+	// SwarmEventType is the event type that swarm generates
+	SwarmEventType = "swarm"
```

Comment:
@allencloud I'm waiting for this PR to make it to `code-review` before doing a PR in engine-api (that's what I tend to do all the time :angel:)